lib/Sema/CSStep.cpp

```diff
@@ -490,6 +490,15 @@ bool DisjunctionStep::shouldStopAt(const DisjunctionChoice &choice) const {
           shortCircuitDisjunctionAt(choice, lastChoice));
 }
 
+bool swift::isSIMDOperator(ValueDecl *value) {
+  auto func = dyn_cast<FuncDecl>(value);
```

Comment:
Maybe check for `!value` should just go here because it makes all of the new `if`s simpler?...

Reply:
Yeah, I can do that. New PR coming shortly, thanks!